modules/rest/src/main/java/org/apache/ignite/internal/rest/recovery/DisasterRecoveryController.java

```diff
@@ -78,6 +80,15 @@ public CompletableFuture<GlobalPartitionStatesResponse> getGlobalPartitionStates
                 .thenApply(DisasterRecoveryController::convertGlobalStates);
     }
 
+    @Override
+    public CompletableFuture<Void> resetPartitions(@Body ResetPartitionsRequest command) {
```

Comment:
What happens if no property specified in the request is passed in the request body? Is there a validation against this?

Reply:
Do you mean null instead of ResetPartitionsRequest? It automatically fails with BAD_REQUEST, "Required Body [command] not specified", no additional validation needed. And table_name/zone_name are checked in the constructor: 
```
        Objects.requireNonNull(zoneName);
        Objects.requireNonNull(tableName);
```